config/fcos/v1_6_exp/schema.go

```diff
@@ -22,6 +22,8 @@ type Config struct {
 	base.Config `yaml:",inline"`
 	BootDevice  BootDevice `yaml:"boot_device"`
 	Grub        Grub       `yaml:"grub"`
+	Selinux     Selinux    `yaml:"selinux"`
+	Systemd     Systemd    `yaml:"systemd"`
```

Comment:
Hmm, why are we overriding the `systemd` key? I think we only need this if we need to change something in how the toplevel `systemd` Butane key gets translated, but that's not the case here (our `selinux` sugar leverages the `systemd` field in the Ignition config, but shouldn't affect the Butane `systemd` schema).